sw/host/vendor/mpsse/support.c

```diff
@@ -183,7 +183,7 @@ int raw_write(struct mpsse_context* mpsse, uint8_t* buf, int size) {
             */
          else if (cmd == mpsse->tx) {
            /* Need to make data out an input to avoid contention on the bus when
-            * the slave sends an ACK */
+            * the recipient sends an ACK */
```

Comment:
This is a vendored file, so it needs to be changed via a patch file in `sw/host/vendor/patches/mpsse` if it's going to be modified. We can open an upstream pull request to try to reduce the amount of local modification we need to make.

Reply:
> This is a vendored file, so it needs to be changed via a patch file in `sw/host/vendor/patches/mpsse` if it's going to be modified. We can open an upstream pull request to try to reduce the amount of local modification we need to make.

Yeah, I forgot to mention there are several vendored-in files that I need to chase down too, including Ibex, ETH, and this one.

Reply:
I'll back this one out and let it be handled upstream.